common/src/main/scala/tsec/common/package.scala

```diff
@@ -17,13 +18,32 @@ package object common {
     override def getMessage: String = cause
   }
 
-
   trait StringNewt {
     type I <: String
 
     val is: Is[I, String]
   }
 
+  implicit final class TSecFloatOps(val v: Float) extends AnyVal {
+    def toBytes: Array[Byte] = ByteBuffer.allocate(4).putFloat(v).array()
+  }
+
+  implicit final class TSecDoubleOps(val v: Double) extends AnyVal {
+    def toBytes: Array[Byte] = ByteBuffer.allocate(8).putDouble(v).array()
+  }
+
+  implicit final class TSecLongOps(val v: Long) extends AnyVal {
+    def toBytes: Array[Byte] = ByteBuffer.allocate(8).putLong(v).array()
+  }
+
+  implicit final class TSecIntOps(val v: Int) extends AnyVal {
+    def toBytes: Array[Byte] = ByteBuffer.allocate(4).putInt(v).array()
+  }
+
+  implicit final class TSecShortOps(val v: Short) extends AnyVal {
```

Comment:
Could these be in the same style as the others? Aka final class, implicit def.

Simply for consistency's sake